prebuilts-buildbot.py

```diff
@@ -40,7 +40,7 @@
 BUILDBOT_PLATFORM_TRIPLES = (
     'x86-linux-debian8',
     'x86_64-linux-debian8',
-    'armv6-android-api9',
+    'armv6-android-api19',
```

Comment:
I'm thinking if the platform triple for android should include both the api and the ndk version